values-kibana.yaml

```diff
@@ -7,10 +7,10 @@ replicas: 1
 resources:
   requests:
     cpu: "100m"
-    memory: "500Mi"
+    memory: "256Mi"
   limits:
     cpu: "1000m"
-    memory: "1Gi"
+    memory: "256Mi"
```

Comment:
Why do you think we need to reduce the limit as well?

Reply:
I did not test these reductions and suppose you verified those are working even for larger sets of logs?